src/keycloak/_version.py

```diff
@@ -21,6 +21,6 @@
 # IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-import pkg_resources
+from importlib import metadata
```

Comment:
This is only available for Python 3.8+. So you need to add a compat shim for Python 3.7.

Reply:
We're dropping support for Python 3.7, as Python 3.7 itself is not supported by Python https://devguide.python.org/versions/, so this is fine :)